src/main/typescript/CookieAuth.tsx

```diff
@@ -32,7 +31,9 @@ const CookieAuth: FC<CookieAuthProps> = ({ isAuthenticated, cookieAuthenticate,
             cookieAuthenticate()
     }, [])
 
-    return (<>{children}</>)
+    return (
+        isAuthenticated ? <>{children}</> : <p>Your session is expired or invalid</p>
```

Comment:
Don't know how to provoke this message on deasy.

Reply:
Actually, when directly accessing the deposit-ui without a valid cookie you first get a page with a red error message: "An error occurred: missing, invalid or expired credentials. Cannot load data from the server."

When then refreshing the page, and empty page with just the message of line 35 appears.